assets/widget.js

```diff
@@ -61,12 +62,41 @@ function show_modal( $modal ) {
 	$('#republication-tracker-tool-modal-content').unbind().click(function(e) {
 		e.stopPropagation();
 	});
+	trapFocus( $modal );
+	$close.focus();
 }
 
-function close_modal( $modal ) {
+function trapFocus( $modal ) {
```

Comment:
Not a criticism of your code, but I couldn't believe there's no easier standards-based way to trap focus inside a specific container! I could not find a better, simpler, solution, but this seems like it should be handleable using native HTML attributes or something.

Reply:
Agreed! I'd like to do some more digging before we roll out similar changes to a wider number of elements -- it seems like the kind of thing that should have a simpler solution, I didn't have any luck finding one yet, either!

Reply:
Seems like `role="dialog" is supposed to be that, but I'm not sure it affects keyboard tabbing

Reply:
Right?! It looks like [the `<dialog>` element](https://developer.mozilla.org/en-US/docs/Web/API/HTMLDialogElement/showModal) might be the way to go here in the future, though I'd like to experiment with it a bit and get confirmation from someone better skilled in accessibility that it covers everything 🙂